packages/widget-message-meet/Gruntfile.js

```diff
@@ -47,6 +58,8 @@ module.exports = function configGrunt(grunt) {
     });
 
   grunt.registerTask('build', [
+    'clean:dist',
+    'env:build',
```

Comment:
Yea, I can remove those `env:build` changes. I changed the start script in the build to use the `webpack.DefinePlugin` method to set it to "production" during the build. Will revert those `env:build` changes.